### PR TITLE
Overhaul quick iframe embedding

### PR DESCRIPTION
**Pull request by dustsigns**
Work in progress

TODOs:

* Small fixes after clarification
* Make sure rendering is still o.k.
* Link this page from the setup guide(s)

### CODE REVIEW

File: docs/first/embed-iframe.rst [hunk 1]
```diff
@@ -1,37 +1,25 @@
...
-
-VerificationUrl: ``https://ropsten.sproof.io/``
+The verifer is usually shipped with a docker container that provides an embeddable iframe. If you prefer a quicker way for testing and/or deploying, this description of the quick setup also provides a customizeable iframe in the end, albeit without the need to set up the docker container yourself.
 
 
 Configuration
----------------------------------
+----------------------------------------
+
+The source of the iframe to embed can be specified by a verifier URL with URL parameters. Depending on the URL and its parameters, you can customize the verifier. The fully URL with parameters is:
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Is an example for the iframe embedding needed here?

File: docs/first/embed-iframe.rst [hunk 2]
```diff
@@ -1,37 +1,25 @@
...
 
 ``{verificationUrl}/#/verify/{lang}?color={color}&btnFontColor={btnFontColor}&profile={profileAddress}&font='{font}'``
 
+The URL (``verificationUrl``) can either be ``https://verify.sproof.io/`` for the mainnet or ``https://ropsten.sproof.io/`` for the testnet. Please refer to :doc:`network-sproof` for a more detailed description about the sproof networks.
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Should a full example URL be added?

File: docs/first/embed-iframe.rst [hunk 3]
```diff
@@ -1,37 +1,25 @@
...
 
+The following URL parameters must be specified:
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Are any or all of the parameters mandatory? Please clarify.

File: docs/first/embed-iframe.rst [hunk 4]
```diff
@@ -1,37 +1,25 @@
...
 
-- ``verificationUrl`` - ``String``: Enter the verificationUrl. There is one for the testnet and one for the mainnet.
-- ``lang`` - ``String``: The verifier page is available in english and german. Use ``de`` or ``en`` here.
-- ``color`` - ``String``: Enter the main color of the verifier ui in hex encoding without #, e.g. ffffff for white.
-- ``btnFontColor`` - ``String``: Color for the Button font. Standard is black.
-- ``profileAddress`` - ``String``: Enter your sproof address here. This will ensure that only document of your profile or your friends are shown als valid on your domain.
-- ``font`` - ``String``: Enter a common font name. If the browser does not support it, it wont work.
+- ``lang`` - ``String`` (mandatory): The verifier page is available in English (value ``en``) and German (``de``).
+- ``color`` - ``String`` (mandatory): Enter the main color of the verifier UI in hexadecimal encoding without #, e.g. ``ffffff`` for white.
+- ``btnFontColor`` - ``String`` (optional): The font color for the Upload button. The default value is black.
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Are there any default values for any of the other parameters?

File: docs/first/embed-iframe.rst [hunk 5]
```diff
@@ -1,37 +1,25 @@
...
+- ``profileAddress`` - ``String`` (mandatory): Enter your sproof profile address here. This ensures that only documents registered by your profile or those of friends you confirmed in the sproof network are shown als valid.
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: What is the easiest way to get the sproof profile address? If it is in the Settings page, I recommend it be linked here in a similar way as in the First steps.

File: docs/first/embed-iframe.rst [hunk 6]
```diff
@@ -1,37 +1,25 @@
...
+- ``font`` - ``String`` (mandatory): The font used for the verifier UI. If the browser displaying the verifier does not support it, a browser-dependent default font may be used.
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Is the described behavior correct?

File: docs/first/embed-iframe.rst [hunk 7]
```diff
@@ -1,37 +1,25 @@
...
-
-VerificationUrl: ``https://verify.sproof.io/``
-
-Testnet
---------------------
-
-The sproof-verifier on the test is hosted on:
-
-VerificationUrl: ``https://ropsten.sproof.io/``
+The verifer is usually shipped with a docker container that provides an embeddable iframe. If you prefer a quicker way for testing and/or deploying, this description of the quick setup also provides a customizeable iframe in the end, albeit without the need to set up the docker container yourself.
```
Comment (dustsigns) [inline docs/first/embed-iframe.rst]: Is a link to the regular setup routine desired here?

File: docs/first/embed-iframe.rst (hunk 1 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: <iframe src="https://verify.sproof.io/#/verify/de?color=000000&btnFontColor=ffffff&profile=0x711d88f76c98a023a7ecf27e167df3f533661626&font='sans-serif'" style="width: 100%;height: 300px;border: none;"></iframe>

File: docs/first/embed-iframe.rst (hunk 2 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: Yes that would be great. 

``https://verify.sproof.io/#/verify/de?color=000000&btnFontColor=ffffff&profile=0x711d88f76c98a023a7ecf27e167df3f533661626&font='sans-serif'``

File: docs/first/embed-iframe.rst (hunk 3 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: No, all parameters are optional. But it is recommended to define the ``profileAddress``.

File: docs/first/embed-iframe.rst (hunk 4 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: Yes, the default values are sproof related, hence profileAddress=sproofsProfile, lang = 'en', font='Catamaran' color='sproof color', btnFontColor='white'

File: docs/first/embed-iframe.rst (hunk 5 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: It is in the settings page and when you click on your profile in the sproof webapp it is shown at the bottom and in the url.

File: docs/first/embed-iframe.rst (hunk 4 above)
Comment (dustsigns) [reply docs/first/embed-iframe.rst]: From the original text, I now changed `profileAddress` to `profile` and `btnFontColor` from black to white. Is this correct?

File: docs/first/embed-iframe.rst (hunk 5 above)
Comment (cle1000) [inline docs/first/embed-iframe.rst]: This parameter should be named as  ``profile`` and not as ``profileAddress``

File: docs/first/embed-iframe.rst (hunk 4 above)
Comment (cle1000) [reply docs/first/embed-iframe.rst]: perfect thanks, that is correct. The invalid parameter name was my fault :)

**Pull request merged**